public_website/src/lib/blocks/testimonyCaousel/testimonyCarouselSlide.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useRef, useState } from 'react'
+import { Slide } from 'pure-react-carousel'
+import styled, { css } from 'styled-components'
+
+import { Typo } from '../../../ui/components/typographies'
+import { APIResponse } from '@/types/strapi'
+import { Pause } from '@/ui/components/icons/Pause'
+import { Play } from '@/ui/components/icons/Play'
+import { getStrapiURL } from '@/utils/apiHelpers'
+
+export type TestimonyCarouselSlideProps = {
+  slideIndex: number
+  image: APIResponse<'plugin::upload.file'> | null
+  title: string
+  description: string
+  url: string
+}
+
+export function TestimonyCarouselSlide({
+  slideIndex,
+  image,
+  title,
+  description,
+  url,
+}: TestimonyCarouselSlideProps) {
+  const testimonyVideoRef = useRef<HTMLVideoElement>(null)
+  const [isTestimonyPlaying, setIsTestimonyPlaying] = useState<boolean>(false)
+  const playTetsimonyVideo = () => {
+    if (testimonyVideoRef.current) {
+      if (!isTestimonyPlaying) {
+        testimonyVideoRef.current.play()
+        setIsTestimonyPlaying(true)
+      } else {
+        testimonyVideoRef.current.pause()
+        setIsTestimonyPlaying(false)
+      }
+    }
+  }
+  return (
+    <Root
+      index={slideIndex}
+      key={title}
+      innerClassName="inner"
+      aria-roledescription="diapositive">
+      <StyledTestimonyWrapper>
+        <button onMouseDown={playTetsimonyVideo}>
+          <span className="visually-hidden">
+            {isTestimonyPlaying ? 'Pause' : 'Lecture'}
+          </span>
+          {!isTestimonyPlaying ? <Play /> : <Pause />}
```

Comment:
Autant rester sur la même condition qu'en ligne 48 
```suggestion
          {isTestimonyPlaying ? <Pause /> : <Play />}
```